langchain4j-redis/src/test/java/dev/langchain4j/store/embedding/redis/RedisEmbeddingStoreIT.java

```diff
@@ -56,4 +63,29 @@ protected EmbeddingStore<TextSegment> embeddingStore() {
     protected EmbeddingModel embeddingModel() {
         return embeddingModel;
     }
+
+    @Test
+    public void testAuthPass() {
+        assertThatNoException().isThrownBy(() -> RedisEmbeddingStore.builder()
+                .host(redis.getHost())
+                .port(redis.getFirstMappedPort())
+                .password("redis-stack")
+                .indexName(randomUUID())
+                .dimension(384)
+                .metadataKeys(createMetadata().toMap().keySet())
+                .build());
+    }
+
+    @Test
+    public void testUnauthorized() {
```

Comment:
you could add this test to the MemoryStore too. And add the wrong password test to the EmbeddingStore.